src/hotspot/share/runtime/abstract_vm_version.hpp

```diff
@@ -130,6 +132,12 @@ class Abstract_VM_Version: AllStatic {
   static uint64_t features()           { return _features; }
   static const char* features_string() { return _features_string; }
 
```

Comment:
IIUC this newline was added during a conflict resolution but there's no such new line in the original changes. Although it will be added in JDK 15+23 anyway...

Not functionally important, just truing to minimize the difference.